frontend/src/components/pages/details/components/DetailsFiles/DetailsFiles.tsx

```diff
@@ -0,0 +1,137 @@
+import { useId, useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import Link from 'next/link';
+
+import { FileFromAttachment } from 'modules/interface';
+import { cn } from 'services/utils/cn';
+import { Minus } from 'components/Icons/Minus';
+import { Plus } from 'components/Icons/Plus';
+import { Paperclip } from 'components/Icons/Paperclip';
+
+interface DetailsDetailsFilesProps {
+  id?: string;
+  className?: string;
+  files?: FileFromAttachment[];
+  asAccordion?: boolean;
+  titleTag?: 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+}
+
+export const DetailsFiles: React.FC<DetailsDetailsFilesProps> = ({
+  className,
+  files = [],
+  asAccordion = false,
+  titleTag: TitleTag = 'h2',
+  ...props
+}) => {
+  const SubTitleTag = TitleTag === 'h2' ? 'h3' : 'h4';
```

Comment:
if the title tag is h5 then the subtitle tag is still h4? 🤔

Reply:
Yes, from a generic point of view, it could be better.
In the concrete case, the `titleTag` can only be 'h2' or 'h3'.